fido/fido.py

```diff
@@ -241,22 +251,32 @@ def handle_timeout_errors(error):
     return deferred
 
 
-def get_agent(reactor, connect_timeout=None):
+def get_agent(reactor, connect_timeout=None, tcp_nodelay=False):
     """Return appropriate agent based on whether an http_proxy is used or not.
 
     :param connect_timeout: connection timeout in seconds
     :type connect_timeout: float
+    :type tcp_nodelay: boolean
     :returns: :class:`twisted.web.client.ProxyAgent` when an http_proxy
         environment variable is present, :class:`twisted.web.client.Agent`
         otherwise.
     """
 
     # TODO: Would be nice to have https_proxy support too.
     http_proxy = os.environ.get('http_proxy')
+
+    pool = None
+
+    if tcp_nodelay:
+        from fido._client import HTTPConnectionPoolOverride
+        pool = HTTPConnectionPoolOverride(reactor, False)
```

Comment:
can you called them by name? It is more explicit for the reader to understand what False stands for.
(
reactor=reactor,
persistent=False,
)